packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.ts

```diff
@@ -125,7 +125,7 @@ export const getLatestTaskDate = (processes: SubscriptionDetailProcess[]) => {
             return dateB.getTime() - dateA.getTime();
         });
 
-    return tasks.length > 0 ? tasks[0].startedAt : '';
+    return tasks && tasks.length > 0 ? tasks[0].startedAt : '';
```

Comment:
Shouldn't tasks always be not null as well here ?

`const tasks = response?.workflows.page || [];`